app/Views/pages/index.php

```diff
@@ -7,8 +7,41 @@
 
 <?php $this->section('content') ?>
 
-<h1>Busca entre tus sitios favoritos</h1>
+<h1 class="title">Busca entre tus sitios favoritos</h1>
```

Comment:
Cambia la clase algo que sea más descriptivo y asociado a la página: 
```suggestion
<h1 class="home-title">Busca entre tus sitios favoritos</h1>
```